### ISSUE TITLE
Pre-release versions show up as an "X" by default

### ISSUE THREAD

**Issue by kotx** [serayuzgur/crates#151]
**Describe the bug**
Pre-release versions will show up as an "X" despite being newer than the latest releases, unless `crates.listPreReleases` is `true`. However, this config setting also lists pre-release versions in the version list, which may not be desired. 

**To Reproduce**
1. Use a RC or beta version of a crate like `rocket 0.5.0-rc.1`
2. See the "X" in Cargo.toml
3. Set `crates.listPreReleases` to `true`
4. See the absence of the "X"

**Expected behavior**
Pre-release versions should not error even when `crates.listPreReleases` is set to `false`.

**Screenshots**

![image](https://user-images.githubusercontent.com/33439542/139600725-12ce51bd-f705-437f-9c1b-cdb1728a3274.png)
![image](https://user-images.githubusercontent.com/33439542/139600726-da73502d-6f2f-4bf7-81c8-7c264e7665ad.png)

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Version 21H2 (22000.282)

**Comment by msdrigg** [comment]
I would agree especially because the settings flag is global. This makes it difficult to use everywhere because some projects I am using pre-releases but others I dont.

Maybe it would be possible to have pre-releases marked with a yellow warning or something similar when the allow-pre-release setting is turned off. (And a similar warning when _not_ using pre-releases when the setting is turned on).

I know this isn’t exactly what you are asking for, but would this resolve your issue?

**Comment by kotx** [comment]
That looks good to me!

**Comment by 0PandaDEV** [comment]
Any updates on this, it really annoys me.

**Comment by serayuzgur** [comment]
we are working on an update, all parsing etc refactored. So please lets deal all with these after the new release. It will be easier to patch things up after refactor.
